### ISSUE TITLE
How to use python-creole python module in my project

### ISSUE THREAD

**Issue by jedie** [jedie/python-creole#24]
*from https://github.com/dae - https://code.google.com/p/python-creole/issues/detail?id=24 :*

I would like to use python-creole python module in my project. It will be an add-on for Anki SRS [http://ankisrs.net/]. That means that my add-on is required to be self-contained. So my question is how can I install python-creole python module into my project's subdirectory? I would probably need to have a license, copyright etc there.

Thanks,
Sergey

**Comment by jedie** [comment]
Just use pip to install it.

**Comment by jedie** [comment]
*from https://code.google.com/p/python-creole/issues/detail?id=24#c2 :*

Sorry, that is not what I am searching for. I thought I could get some advice here.

Hopefully, I think I have found a possible solution for my issue:
python setup.py --no-user-cfg install_lib --install-dir=<installation-directory>

But it does not put the license, authors etc files into an installation directory. I found that probably every file contains copyright statement. Is it okay?
